biz.aQute.bndlib/src/aQute/bnd/service/result/Ok.java

```diff
@@ -170,4 +170,13 @@ public V orElseGet(Supplier<? extends V> orElseSupplier) {
 	public String toString() {
 		return String.format("Ok(%s)", value);
 	}
+
+	@Override
+	public void accept(ConsumerWithException<? super V> ok, ConsumerWithException<? super E> err) {
```

Comment:
Need to add `requireNonNull(err);` for consistent behavior regardless of whether the Result is Ok or Err.